### ISSUE TITLE
Execution stages aren't visible after the execution is finished

### ISSUE THREAD

**Issue by tkonieczny** [kubeshop/testkube#3029]
**Describe the bug**
Test execution stages aren't visible after the execution is finished.

**To Reproduce**
1. Start the test execution
2. Check output (stages are visible)
![Screenshot_20230113_141551](https://user-images.githubusercontent.com/1473096/212330121-dfd4c5c1-6eff-4cdf-a402-fbb77716ae2f.png)

3. Wait for the execution to finish
4. Stages aren't visible anymore
![Screenshot_20230113_141609](https://user-images.githubusercontent.com/1473096/212330135-2ac449f2-b850-4954-9cbf-f50536875960.png)

Comment:
Incorrect test result visible at second screenshot isn't related - it's in separate issue here: #3023

**Comment by vLia** [comment]
This is related to https://github.com/kubeshop/testkube/issues/2396

The problem is that the output of the actual test run is saved only after the execution is finished. Will continue the work here, and close the other ticket then :)

**Comment by vLia** [comment]
Hello @tkonieczny,

Just released a fix in Testkube version 1.8.33, helm-charts 1.8.118. Logs should be visible now after a new execution is done.

**Comment by tkonieczny** [comment]
Logs are now visible, but looks like the test output is now duplicated - there is additional test output after `Execution succeded`, and `Test run successful`
![Screenshot_20230123_163533](https://user-images.githubusercontent.com/1473096/214080984-29c0bb35-8ee8-4476-845f-d840cf9612a3.png)
https://demo.testkube.io/tests/executions/k6-executor-smoke/execution/63cea89922e43b404a0bae50
or, another one:
https://demo.testkube.io/tests/executions/maven-executor-smoke-jdk18/execution/63cea5b622e43b404a0bae4e

**Comment by vLia** [comment]
@tkonieczny The logs are copied over as they are shown in the pods. In k6 the results are printed out twice:  once by the test runner, and once by Testkube printing out the result as the last line. I suggest we address these on a case-by-case basis inside each executor, as it is not happening for all of them

**Comment by vLia** [comment]
Tracking the double print of the results in https://github.com/kubeshop/testkube/issues/3240
